### PR TITLE
add option flag, batch operations can return one res

### PR DESCRIPTION
**Pull request by eemjwu**
## Summary

batch operation add option flag to return one result, reduce the size of rpc package

## Solution Description

1. add RETURN_ONE_RES in ObTableOptionFlag

### CODE REVIEW

File: src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableBatchOps.java [hunk 1]
```diff
@@ -131,6 +132,19 @@ public boolean isAtomicOperation() {
         return atomicOperation;
     }
 
+    @Override
+    public void setReturnOneRes(boolean returnOneRes) {
```
Comment (WeiXinChan) [inline src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableBatchOps.java:136]: setReturnOneResult maybe better

File: src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableBatchOps.java [hunk 2]
```diff
@@ -131,6 +132,19 @@ public boolean isAtomicOperation() {
...
+        this.returnOneRes = returnOneRes;
+    }
+
+    /*
+     * Is atomic operation.
```
[the remark this answers is not in the document]
Comment (eemjwu) [reply src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableBatchOps.java]: done

File: src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableBatchOps.java (hunk 1 above)
Comment (eemjwu) [reply src/main/java/com/alipay/oceanbase/rpc/table/AbstractTableBatchOps.java]: done

File: src/main/java/com/alipay/oceanbase/rpc/ObClusterTableBatchOps.java [hunk 3]
```diff
@@ -175,4 +175,10 @@ public void setAtomicOperation(boolean atomicOperation) {
         super.setAtomicOperation(atomicOperation);
         tableBatchOps.setAtomicOperation(atomicOperation);
     }
+
+    @Override
+    public void setReturnOneRes(boolean returnOneRes) {
```
[the remark this answers is not in the document]
Comment (eemjwu) [reply src/main/java/com/alipay/oceanbase/rpc/ObClusterTableBatchOps.java]: done

**Comment by WeiXinChan** [comment]
LGTM

**Pull request closed without merging**